### PR TITLE
Updated dashboard components including ImageCarousel, Poll and RecentForumPosts + testing

### PR DESCRIPTION
**Pull request by ChloeKVB**
- updated ImageCarousel component .jsx and .css
- updated Poll component .jsx and .css
- updated RecentForumPosts component including .jsx and .css
- testing for ImageCarousel and Poll

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import BroadcastMessages from "./components/dashboard/Broadcast.jsx";
 import ImageCarousel from "./components/dashboard/ImageCarousel.jsx";
 // import "./components/dashboard/ImageCarousel.css";
 import ForumTopicThreads from "./components/forum/ForumTopicThreads";
-import RecentForumPosts from "./components/dashboard/RecentForumPosts";
+import RecentForumPosts from "./components/dashboard/RecentForumPosts"; import Poll from "./components/dashboard/Poll";
```
Comment (cactusballs) [inline client/src/App.jsx:8]: should these be on separate lines?

File: client/src/components/dashboard/Poll.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
+.Poll-card {
```
Comment (cactusballs) [inline client/src/components/dashboard/Poll.css:1]: there doesn't seem to be a ClassName called Poll-card, I don't think this css selector is doing anything at the minute.

File: client/src/components/dashboard/Poll.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Card from "./Card";
+import "./Poll.css";
+import { BsChatLeftHeart } from "react-icons/bs";
+
+const Poll = ({ pollId }) => {
```
Comment (cactusballs) [inline client/src/components/dashboard/Poll.jsx:7]: Is the poll supposed to have functionality or is that coming later?

File: client/src/components/dashboard/Poll.test.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (cactusballs) [inline client/src/components/dashboard/Poll.test.js:1]: this currently fails when ran, still working on it?

File: client/src/components/dashboard/RecentForumPosts.jsx [hunk 5]
```diff
@@ -1,5 +1,6 @@
 import "./RecentForumPosts.css";
```
Comment (cactusballs) [inline client/src/components/dashboard/RecentForumPosts.jsx:1]: This also doesn't appear to fetch anything from the DB

**Review by cactusballs — COMMENTED** [review]
Heyyo Chloe! So much work, well done pal! 

There's a few comments as I don't seem to be getting functionality on my machine but are you on yours? Also in the poll card CSS there seems to be a mismatch! 

Happy to have a huddle if I need to explain anything etc :) 

Thanks so much!

File: package.json [hunk 6]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (cactusballs) [inline package.json:1]: remove this file

**Review by cactusballs — CHANGES_REQUESTED** [review]
remove unnecessary JSON

File: client/src/App.jsx (hunk 1 above)
Comment (ChloeKVB) [reply client/src/App.jsx]: moved to a new line

File: client/src/components/dashboard/Poll.css (hunk 2 above)
Comment (ChloeKVB) [reply client/src/components/dashboard/Poll.css]: fixed

File: client/src/components/dashboard/Poll.jsx (hunk 3 above)
Comment (ChloeKVB) [reply client/src/components/dashboard/Poll.jsx:6]: discussed

File: client/src/components/dashboard/Poll.test.js (hunk 4 above)
Comment (ChloeKVB) [reply client/src/components/dashboard/Poll.test.js:1]: fixed issue

File: client/src/components/dashboard/RecentForumPosts.jsx (hunk 5 above)
Comment (ChloeKVB) [reply client/src/components/dashboard/RecentForumPosts.jsx:1]: fixed issue

File: package.json (hunk 6 above)
Comment (ChloeKVB) [reply package.json]: removed file

**Pull request merged**